python/cuvs/cuvs/neighbors/brute_force/brute_force.pyx

```diff
@@ -256,3 +257,73 @@ def search(Index index,
         ))
 
     return (distances, neighbors)
+
+
+@auto_sync_resources
+def save(filename, Index index, bool include_dataset=True, resources=None):
```

Comment:
Can you please add the Python and C++ brute force serialization APIs to the docs?